x-pack/legacy/plugins/fleet/server/repositories/policies/default.ts

```diff
@@ -4,28 +4,45 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { IngestPlugin, PoliciesRepository as PoliciesRepositoryType } from './types';
+import {
+  IngestPolicyLib,
+  IngestOutputLib,
+  PoliciesRepository as PoliciesRepositoryType,
+} from './types';
 import { FrameworkUser } from '../../adapters/framework/adapter_types';
 
 export class PoliciesRepository implements PoliciesRepositoryType {
-  constructor(private readonly plugin?: IngestPlugin) {}
+  constructor(
+    private readonly ingestPolicyLib?: IngestPolicyLib,
+    private readonly ingestOutputLib?: IngestOutputLib
+  ) {}
 
   /**
    * Return a full policy
    *
    * @param id
    */
   async getPolicyOutputByIDs(user: FrameworkUser, ids: string[]) {
-    if (this.plugin) {
-      return await this.plugin.getPolicyOutputByIDs(user, ids);
+    if (this.ingestOutputLib) {
+      return await this.ingestOutputLib.getByIDs(
+        {
+          kind: 'internal',
+        },
+        ids
+      );
     }
 
     return [];
   }
 
   async get(user: FrameworkUser, id: string) {
-    if (this.plugin) {
-      return await this.plugin.getPolicyById(user, id);
+    if (this.ingestPolicyLib) {
+      return await this.ingestPolicyLib.get(
+        {
+          kind: 'internal',
```

Comment:
what does this `kind: 'internal'` flag mean? is it the user?

Reply:
It's using the internal kibana user to make requests to elastic